src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -335,11 +312,11 @@ private void dryrunStandardizeBatch(List<Long> pIdGroup, int nonMatchingCmpds, i
 				stndznCompound.setNewMolWeight(null);
 			}
 
-			if (newMolWeight == null || stndznCompound.getParent().getMolWeight() == null) {
+			if (newMolWeight == null || parent.getMolWeight() == null) {
```

Comment:
I was noticing this was creating a new transaction every time it was fetching parent then mol weight so I used the already in memory parent instead of having to fetch it fresh every time.